layout/_partial/post/meta.ejs

```diff
@@ -6,8 +6,14 @@
 		    <%= post.date.locale(page.lang).format(__('date_format')) %>
     	<% } %>
     </time>
+    <% if (post.updated) { %>
+    <time itemprop="dateModified" datetime="<%= post.updated.format() %>" />
+    <% } %>
     <% if ((post.categories) && (post.categories.length > 0)) { %>
         <span><%= __('post.categorized_in')  %> </span>
         <%- partial('category', {categories: post.categories})%>
     <% } %>
+    <div itemprop="author" itemscope itemtype="http://schema.org/Person">
```

Comment:
the head tag does not apply to "article" schema.org. Again, Google´s structured testing tool will complain that every single entity of the page be marked even it being redundant. As per webmaster discussions there, in the index page for instance, every single excerpt  is an "BlogPosting" entity and a BlogPosting must explicitly define its Author (@person), Publisher (@Organization for google´s purpose, so I left it out for now because it yelds other errors if set to @Person). and image (I set it optionally if the posting image is defined by user - as theme already do in fact).